mandatory/2-function-creation.js

```diff
@@ -16,7 +18,11 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  num = parseFloat(num.toFixed(2));
+  let formattedNum = (num > 100) ? `100%` : `${num}%`;
```

Comment:
Well done! This is a very nice and tidy use of conditional (ternary) operator instead of if statement.